recipes/bonjour/system/conanfile.py

```diff
@@ -0,0 +1,27 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class BonjourSystemConan(ConanFile):
+    name = "bonjour"
+    provides = "mdnsresponder"
+    version = "system"
+    description = "Conan package for Apple's Bonjour"
+    topics = ("Bonjour", "DNS-SD", "mDNS")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://developer.apple.com/bonjour/"
+    license = "Apache-2.0", "BSD-3-Clause"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def validate(self):
+        if self.settings.os != "Windows" or tools.cross_building(self):
```

Comment:
```suggestion
        if self.settings.os != "Windows" or (hasattr(self, "settings_build") and tools.cross_building(self, skip_x64_x86=True)):
```
the problem is that ConanCenterContinuousIntegration checks the valid configurations from Linux, so when it checks windows it thinks it is cross compiling